### ISSUE TITLE
Transparency not working for OBST

### ISSUE THREAD

**Issue by rmcdermo** [firemodels/fds#8053]
According to the user guide, we should be able to set TRANSPARENCY for an OBST.  But this does not seem to work with current FDS-SMV (not sure which is to blame).  Assigning transparency to a SURF, then to the OBST works.  Here is a case with just an OBST, and in this case TRANSPARENCY is not working.

[obst_transparency.fds.txt](https://github.com/firemodels/fds/files/4231741/obst_transparency.fds.txt)

**Comment by gforney** [comment]
I'll take a look

**Comment by gforney** [comment]
sorry for the delay - I somehow downloaded the wrong test case (which I couldn't reproduce the problem with).  After downloading the right test case, I was able to reproduce the problem.  

It looks like fds is not copying transparency info to the smv file when a COLOR or RGB keyword is not specified on an OBST line.  When I added RGB=0,0,255 to the &OBST lines in your test case worked.

For now add RGB=r,g,b where r/g/b ranges from 0 to 255 to your &OBST lines and transparency should work. (or COLOR='some color')

I'll look at FDS to see if I can figure out &OBST output to the .smv file (this was written 20 years ago!) so transparency works with default colors.
